qcfractal/alembic/versions/a2f76bb7be65_updates_collections_table_with_owner_.py

```diff
@@ -20,6 +20,7 @@ def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column('collection', sa.Column('description', sa.String(), nullable=True))
     op.add_column('collection', sa.Column('owner', sa.String(length=100), nullable=True))
+    op.excecute("UPDATE collection SET owner=default")
```

Comment:
@dgasmith @doaa-altarawy Could one of you please verify that (i) this is correct SQL and (ii) this change to the migration won't mess with our current partially migrated state on molssi10.